pkg/kopia/command/storage/secret_utils.go

```diff
@@ -0,0 +1,44 @@
+package storage
+
+type LocType string
+
+const (
+	LocTypeS3        LocType = "s3"
+	LocTypeGCS       LocType = "gcs"
+	LocTypeAzure     LocType = "azure"
+	LocTypeFilestore LocType = "filestore"
+)
+
+const (
+	bucketKey        = "bucket"
+	endpointKey      = "endpoint"
+	prefixKey        = "prefix"
+	regionKey        = "region"
+	skipSSLVerifyKey = "skipSSLVerify"
+	typeKey          = "type"
+)
+
+func bucketName(m map[string]string) string {
```

Comment:
I would suggest renaming these functions as - `getBucketNameFromMap`. Same for other functions as well